plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/SubversionClientServiceImpl.java

```diff
@@ -117,12 +117,14 @@ private String getBaseUrl() {
     }
 
     @Override
-    public Promise<CLIOutputResponse> copy(Path project, Path source, Path destination, String comment) {
+    public Promise<CLIOutputResponse> copy(Path project, Path source, Path destination, String comment, Credentials credentials) {
```

Comment:
here https://github.com/eclipse/che/pull/2672/files#diff-4718e5d14a9383c2496b7b13dcca43ebR96 you mark credentials as Nullable but this implementation doesn't support null value. So you can get NPE here